src/apps/content-editor/src/app/components/Editor/Editor.js

```diff
@@ -152,8 +152,10 @@ export default memo(function Editor({
           <h1 className={styles.Display}>No fields have been added</h1>
           <h2 className={styles.SubHead}>
             Use the{" "}
-            <AppLink to={`/schema/${model.ZUID}`}>Schema Builder</AppLink> to
-            define your items content
+            <AppLink to={`/schema/${model?.ZUID || ""}`}>
```

Comment:
I do not think we should render a non-functional link if there is no model data present

Reply:
Do you think it's better to just show an error message if there's no model ZUID? Since right now, what I did is redirect the users to /schema in case the ZUID is not present.

Reply:
Makes sense to just use the modelZUID from the url instead of the model data. I will revise it on my next commit.